README.md

```diff
@@ -42,6 +42,12 @@ To install the components, no additional configuration is needed for internal sy
 
 ## Developing
 
+You can optionally install [nvm](https://github.com/nvm-sh/nvm) to use the correct node version. After installing nvm on your machine, install the suitable node version:
+
+```console
+nvm i
+```
+
```

Comment:
I'm not sure if this needs adding to the readme? It's more for development in general rather than an instruction that is specific to working in this repo

Reply:
I can remove. Just did the same I did for nasa (https://github.com/nasa/openmct/pull/6938). Hard to spot an nvmrc file, but if its in the readme, it becomes clear

Reply:
Ok that makes sense - happy for you to leave it in 👍